101-sql-database-Import/azuredeploy.parameters.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "serverName": {
+      "value": "GEN-UNIQUE"
+    },
+    "administratorLogin": {
+      "value": "GEN-UNIQUE"
+    },
+    "administratorLoginPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "storageAccountType": {
```

Comment:
remove the params that have defaultValues in the template

Reply:
Done

Reply:
Done